### ISSUE TITLE
Preview obscures menus in video recording setup

### ISSUE THREAD

**Issue by OperationXen** [wowarenalogs/wowarenalogs#297]
When setting up the video recording config I started by setting the VOD folder, now when I come to select audio, resolution or capture mode the menus appear behind the video preview (which is currently blank), this prevents me from selecting the options I want

![image](https://github.com/wowarenalogs/wowarenalogs/assets/12632503/8461b413-1576-468c-9162-a7c46541438e)

**Comment by prsauer** [comment]
Yeah fair callout here - the preview implementation is very much a hack atm so wonky stuff can happen

You can reset its position to a better one by clicking in the preview window ;)
